src/objects/media/media.stories.mdx

```diff
@@ -139,3 +141,20 @@ In this example, the object is [a Calendar Date component](/docs/components-cale
     {calendarDateDemo}
   </Story>
 </Canvas>
+
+## Jaunty
+
+By adding the `o-media--jaunty` class, the media object will have a fun little rotation applied to it.
+In this example, we're using it for a Promo.
+
+You may want to add some of our [spacing utility classes](/docs/utilities-spacing--margin) to the media object to compensate for the rotation.
+In this example, we've used `u-space-inline-end-1` and `u-space-inline-start-n4` to add a bit more space between the image and the text, and keep the image from hitting the left side of the viewport on small screens.
```

Comment:
It feels strange to me to create a modifier but then say you probably need utility classes. Any reason not to make adjustments using the modifier class?

Reply:
That sounds a bit nicer to me.

Reply:
Done!